vignettes/s3-vector.Rmd

```diff
@@ -627,15 +627,17 @@ decimal2(10, c(0, 5, 99))
 
 ## Equality and comparison
 
-vctrs provides two "proxy" generics that lets you control how your class determines equality and ordering:
+vctrs provides thee "proxy" generics. Two of those let you control how your class determines equality and ordering:
 
-* `vec_proxy()` returns the actual data of a vector. It is used in most vctrs primitives, including equality comparison. It underpins `==`, `!=`, `unique()`, `anyDuplicated()`, and `is.na()`.
+* `vec_proxy()` returns the actual data of a vector. This is useful when you store the data in a field of your class. Most of the time, you shouldn't need to implement `vec_proxy()`.
```

Comment:
This bulleted list needs a bit more reorganisation to read well. Maybe something like:

---

vctrs provides thee "proxy" generics. Two of those let you control how your class determines equality and ordering:

* `vec_proxy_equal()` ...
* `vec_proxy_compare()` ...

By default, `vec_proxy_equal()` and `vec_proxy_compare()` just call `vec_proxy()`.

* `vec_proxy()` ...

You should only implement ....